rcv/nes56_cv_manager.py

```diff
@@ -23,6 +24,7 @@ def __init__(self, conf):
         self._roborio_ip = self._conf.get('rcv_server', 'roborio_ip')
         self._roborio_port = int(self._conf.get('rcv_server', 'roborio_port'))
         self._socket_timeout = int(self._conf.get('rcv_server', 'socket_timeout'))
+        self._connection_status = 'not connected'
```

Comment:
this is a wrong approach"
1 - using a string variable to indicate a state (better use a boolean)
2 - this variable does not own the state .... meaning if you forget to change it (a bug) somewhere the state will not be reflected correctly.

Please read https://stackoverflow.com/questions/35861484/how-to-know-the-if-the-socket-connection-is-closed-in-python especially answer 3 (use fileno on the socket to check whether its connected).
And than all you need to do when sending the data is to check it and if not connected try to reconnect (once) before sending the data.